### PR TITLE
Fixed not ligthing the sequencer "Save" button after Ins or Del a combination https://github.com/GrandOrgue/grandorgue/issues/2024

### PR DESCRIPTION
**Pull request by oleg68**
Resolves: #2024

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+- Fixed not ligthing the sequencer "Save" button after Ins or Del a combination https://github.com/GrandOrgue/grandorgue/issues/2024
```
Comment (larspalo) [inline CHANGELOG.md:1]:
```suggestion
- Fixed the sequencer "Save file" button not lightening after inserting or deleting a combination https://github.com/GrandOrgue/grandorgue/issues/2024
```

**Review by larspalo — APPROVED** [review]
Seems to work. I suggested another wording, with hopefully better English, for the CHANGELOG.md, but do as you wish with that suggestion.

File: CHANGELOG.md (hunk 1 above)
Comment (oleg68) [reply CHANGELOG.md]: Fixed

**Comment by oleg68** [comment]
@rousseldenis could you approve this PR?

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**